logging/logging.go

```diff
@@ -126,6 +126,15 @@ func caller(depth int) (file string, line int) {
 	return
 }
 
+// toSafeJSONString converts to valid JSON string, as the original string may contain '\\', '\n', '\r', '\t' and so on.
+func toSafeJSONString(s string) []byte {
+	if data, err := json.Marshal(s); err == nil {
+		return data
+	} else {
+		return []byte(`"JSON Marshal Error"`)
```

Comment:
Maybe we just return `"` + s + `"`  when occurs error?

Reply:
Agree. Addressed.